tests/subworkflows/local/grohmm/test.yml

```diff
@@ -0,0 +1,14 @@
+- name: subworkflow grohmm
+  command: nextflow run ./tests/subworkflows/local/grohmm/ -entry test_grohmm -c tests/config/nextflow.config -c tests/subworkflows/local/grohmm/nextflow.config --transcript_identification grohmm
+  # FIXME Tests don't work because of bin/ scripts
+  # tags:
+  # - grohmm
+  # TODO md5 sum these
```

Comment:
sounds like it could be an issue

Reply:
https://github.com/nextflow-io/nextflow/issues/1868
It is adding them as comments though!